src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -234,11 +270,25 @@ public DeploymentOptions()
             foreach (var deployOptionsProp in deploymentOptionsProperties)
             {
                 var prop = options.GetType().GetProperty(deployOptionsProp.Name);
+                var attribute = prop.GetCustomAttributes(typeof(DescriptionAttribute), true)[0];
+                var description = (DescriptionAttribute)attribute;
 
                 // Note that we set excluded object types here since dacfx has this value as null;
                 if (prop != null && deployOptionsProp.Name != "ExcludeObjectTypes")
                 {
-                    deployOptionsProp.SetValue(this, prop.GetValue(options));
+                    if (deployOptionsProp.Name == "DoNotDropObjectTypes")
```

Comment:
Please add a comment about why you're ignoring this one. 

And generally we should avoid using magic strings like this - please make a const in a common location somewhere for these special names.